prog/dftb+/lib_type/typegeometryhsd.F90

```diff
@@ -299,6 +326,16 @@ subroutine readTGeometryGen_help(node, geo, text)
       end do
     end if
 
+    if (geo%tHelical) then
+      allocate(geo%latVecs(3, 1))
+      call getNextToken(text, geo%latVecs(:, 1), iStart, iErr)
```

Comment:
Should probably get a new record here first, otherwise I can scatter the helical boundary conditions over three lines. Something like:
```fortran
iEnd = nextLine(text, iStart)
call getNextToken(text(:iEnd), geo%latVecs(:, 1), iStart, iErr)
```